go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/Azure/azure-storage-azcopy
 
 require (
+	cloud.google.com/go/storage v1.8.0
 	github.com/Azure/azure-pipeline-go v0.2.1
```

Comment:
We should look into the open source licenses for this and google.golang.org/api, and look into the appropriate steps to get these components approved. I've pinged you on Teams with the appropriate place to look.